client/settings/payment-settings/promotional-banner/oc-promotion-banner.js

```diff
@@ -97,7 +97,7 @@ export const OCPromotionBanner = ( {
 						{ __( '', 'woocommerce-gateway-stripe' ) }
 						{ interpolateComponents( {
 							mixedString: __(
-								"Optimize your checkout for more sales by automatically displaying the most relevant payment methods for each customer. {{docLink}}Learn more{{/docLink}} about Stripe's Optimized Checkout Suite.",
+								"Optimize your checkout for more sales by dynamically displaying the most relevant payment methods you've enabled for each customer. {{docLink}}Learn more{{/docLink}} about Stripe's Optimized Checkout Suite.",
```

Comment:
I believe this should be "checkout experience" per the source docs.
```suggestion
								"Optimize your checkout experience for more sales by dynamically displaying the most relevant payment methods you've enabled for each customer. {{docLink}}Learn more{{/docLink}} about Stripe's Optimized Checkout Suite.",
```

It also looks like we need to change `conversions` to `conversion` in the title element.